src/app/qualification/qualification.component.html

```diff
@@ -0,0 +1,36 @@
+<h1>Hiiiii</h1>
```

Comment:
This is ridiculous. Check for these stuff before commits and pull requests and Remove them